js/modules/timeline.src.js

```diff
@@ -0,0 +1,431 @@
+                pointMarkerOptions.states[state];
+
+            radius = pick(
+                pointStateOptions && pointStateOptions.radius,
+                seriesStateOptions && seriesStateOptions.radius,
+                radius + (
+                    seriesStateOptions && seriesStateOptions.radiusPlus ||
+                    0
+                )
+            );
+        }
+
+        point.hasImage = symbol && symbol.indexOf('url') === 0;
+
+        attribs = {
+            x: Math.floor(point.plotX) - (width / 2) - (radius / 2),
+            y: point.plotY - (height / 2) - (radius / 2),
+            width: width + radius,
+            height: height + radius
+        };
+
+        return attribs;
+
+    },
+    bindAxes: function () {
+        var timelineXAxis = {
+                gridLineWidth: 0,
+                lineWidth: 0,
+                min: 0,
+                dataMin: 0,
+                minPadding: 0,
+                max: 100,
+                dataMax: 100,
+                maxPadding: 0,
+                title: null,
+                tickPositions: []
+            },
+            timelineYAxis = {
+                gridLineWidth: 0,
+                min: 0.5,
+                dataMin: 0.5,
+                minPadding: 0,
+                max: 1.5,
+                dataMax: 1.5,
+                maxPadding: 0,
+                title: null,
+                labels: {
+                    enabled: false
+                }
+            };
+        Series.prototype.bindAxes.call(this);
+        extend(this.xAxis.options, timelineXAxis);
+        extend(this.yAxis.options, timelineYAxis);
+    },
+    translate: function () {
+        var series = this,
+            points = series.points,
+            xMap;
+
+        if (points) {
+            xMap = series.mappedXData = this.getMappedXData();
+
+            each(points, function (point, i) {
+                point.applyOptions({
+                    x: xMap[i]
+                });
+            });
+        }
+        Series.prototype.translate.call(this);
+    }
+}, {
+    init: function () {
+        var args = arguments,
+            point = Point.prototype.init.apply(this, args),
+            xMap = args[0].mappedXData;
+
+        point.applyOptions({
+            x: xMap[args[2]],
+            y: 1
+        });
+
+        return point;
+    },
+    getConnectorPath: function () {
+        var point = this,
+            chart = this.series.chart,
+            xAxisLen = point.series.xAxis.len,
+            yAxisLen = point.series.yAxis.len,
+            inverted = chart.inverted,
+            dl = point.dataLabel,
+            targetDLPos = dl.targetPosition,
+            coords = {
+                x1: inverted ? point.plotY : point.plotX,
+                y1: inverted ? xAxisLen - point.plotX : point.plotY,
+                x2: inverted ? targetDLPos.x || dl.x : point.plotX,
+                y2: inverted ? xAxisLen - point.plotX : targetDLPos.y || dl.y
```

Comment:
`point.plotX` set as variable

Reply:
Consider readability

Reply:
Changed to two separate objects. One is overwritten by another, when the chart is inverted.